### PR TITLE
Fix LGTM errors in examples/

### PR DESCRIPTION
**Pull request by rsimha-amp**
Fixes the following LGTM errors:

- [examples/gfycat.amp.html#L28](https://lgtm.com/projects/g/ampproject/amphtml/snapshot/a4d5dedaa4349a630d64ac4ffa41ed08b90f7efe/files/examples/gfycat.amp.html#L28)
- [examples/ima-video.amp.html#L31](https://lgtm.com/projects/g/ampproject/amphtml/snapshot/a4d5dedaa4349a630d64ac4ffa41ed08b90f7efe/files/examples/ima-video.amp.html?sort=name&dir=ASC&mode=heatmap&excluded=false#L31)
- [examples/forms.amp.html#L259](https://lgtm.com/projects/g/ampproject/amphtml/snapshot/a4d5dedaa4349a630d64ac4ffa41ed08b90f7efe/files/examples/forms.amp.html?sort=name&dir=ASC&mode=heatmap&excluded=false#L259)
- [examples/forms.amp.html#L323](https://lgtm.com/projects/g/ampproject/amphtml/snapshot/a4d5dedaa4349a630d64ac4ffa41ed08b90f7efe/files/examples/forms.amp.html?sort=name&dir=ASC&mode=heatmap&excluded=false#L323)
- [examples/forms.amp.html#L518](https://lgtm.com/projects/g/ampproject/amphtml/snapshot/a4d5dedaa4349a630d64ac4ffa41ed08b90f7efe/files/examples/forms.amp.html?sort=name&dir=ASC&mode=heatmap&excluded=false#L518)
- [examples/ads.amp.html#L627](https://lgtm.com/projects/g/ampproject/amphtml/snapshot/a4d5dedaa4349a630d64ac4ffa41ed08b90f7efe/files/examples/ads.amp.html?sort=name&dir=ASC&mode=heatmap&excluded=false#L627)

Partial fix for #12597

### CODE REVIEW

**Comment by rsimha-amp** [comment]
Errors fixed: https://lgtm.com/projects/g/ampproject/amphtml/rev/pr-906b3c620f0097d00d573669405361f62576ce4a

File: examples/ads.amp.html [hunk 1]
```diff
@@ -622,7 +622,7 @@ <h2 id="custom">Custom leaderboard</h2>
       data-slot="1"
       data-url="/examples/custom.ad.example.json">
     <template type="amp-mustache" id="amp-template-id-leader">
-      <a href="{{href}}" target='_blank'>
+      <a href="{{href}}" target='_blank' rel="noopener noreferrer">
```
Comment (choumx) [inline examples/ads.amp.html]: OOC why are these problematic?

Comment (rsimha-amp) [reply examples/ads.amp.html]: It's a security risk. See https://lgtm.com/rules/1806963085/

**Pull request merged**